android/src/main/java/com/reactlibrary/wootric/RNWootricModule.java

```diff
@@ -3,15 +3,23 @@
 import static com.facebook.react.bridge.UiThreadUtil.runOnUiThread;
 
 import android.app.Activity;
+import android.net.Uri;
 import android.util.Log;
 
+import com.facebook.react.modules.core.DeviceEventManagerModule;
+
+import com.facebook.react.bridge.Arguments;
+import com.facebook.react.bridge.Callback;
 import com.facebook.react.bridge.ReactApplicationContext;
 import com.facebook.react.bridge.ReactContextBaseJavaModule;
 import com.facebook.react.bridge.ReactMethod;
 import com.facebook.react.bridge.ReadableMap;
 import com.facebook.react.bridge.ReadableMapKeySetIterator;
 import com.facebook.react.bridge.ReadableType;
+import com.facebook.react.bridge.WritableMap;
+
```

Comment:
Can you check which imports are actually needed still? I think `Callback` and `Uri` aren't, right?

Reply:
You're right. Will clean up when the PR is approved. (Callback is from an older version, URI from rebase)